upstream.go

```diff
@@ -1514,6 +1520,8 @@ func (uc *upstreamConn) SendMessageLabeled(downstreamID uint64, msg *irc.Message
 		}
 		msg.Tags["label"] = irc.TagValue(fmt.Sprintf("sd-%d-%d", downstreamID, uc.nextLabelID))
 		uc.nextLabelID++
+	} else if !uc.caps["message-tags"] {
+		msg.Tags = nil
```

Comment:
Can we define `func (uc *upstreamConn) SendMessage()` and do this juggling there?